third_party/2and3/werkzeug/exceptions.pyi

```diff
@@ -35,6 +35,13 @@ class BadHost(BadRequest): ...
 class Unauthorized(HTTPException):
     code: int
     description: str
+    www_authenticate: Optional[Iterable[object]]
+    def __init__(
+        self,
+        description: Optional[Text] = ...,
+        response: Optional[Response] = ...,
+        www_authenticate: Union[None, Tuple[object], List[object], object] = ...,
```

Comment:
This union could technically be reduced to just `object`, but I like the fact that it expresses the different cases handled by the implementation better.

Reply:
`Tuple[object, ...]` surely?